File: csharp/src/main/CsharpDotNet2/spoonacular/Model/InlineResponse20057.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace spoonacular.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class InlineResponse20057 {
    /// <summary>
    /// Gets or Sets Suggests
    /// </summary>
    [DataMember(Name="suggests", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "suggests")]
    public InlineResponse20057Suggests Suggests { get; set; }

    /// <summary>
    /// Gets or Sets Words
    /// </summary>
    [DataMember(Name="words", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "words")]
    public List<Object> Words { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class InlineResponse20057 {\n");
      sb.Append("  Suggests: ").Append(Suggests).Append("\n");
      sb.Append("  Words: ").Append(Words).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart/com/spoonacular/client/model/inline_response20057.dart
part of openapi.api;

class InlineResponse20057 {
  
  InlineResponse20057Suggests suggests = null;
  
  List<Object> words = [];
  InlineResponse20057();

  @override
  String toString() {
    return 'InlineResponse20057[suggests=$suggests, words=$words, ]';
  }

  InlineResponse20057.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    if (json['suggests'] == null) {
      suggests = null;
    } else {
      suggests = new InlineResponse20057Suggests.fromJson(json['suggests']);
    }
    if (json['words'] == null) {
      words = null;
    } else {
      words = Object.listFromJson(json['words']);
    }
  }

  Map<String, dynamic> toJson() {
    Map <String, dynamic> json = {};
    if (suggests != null)
      json['suggests'] = suggests;
    if (words != null)
      json['words'] = words;
    return json;
  }

  static List<InlineResponse20057> listFromJson(List<dynamic> json) {
    return json == null ? new List<InlineResponse20057>() : json.map((value) => new InlineResponse20057.fromJson(value)).toList();
  }

  static Map<String, InlineResponse20057> mapFromJson(Map<String, dynamic> json) {
    var map = new Map<String, InlineResponse20057>();
    if (json != null && json.isNotEmpty) {
      json.forEach((String key, dynamic value) => map[key] = new InlineResponse20057.fromJson(value));
    }
    return map;
  }
}

